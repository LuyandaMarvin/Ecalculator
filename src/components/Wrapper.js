import React from 'react'
import logo from '../logo.png';

const Wrapper = ({ children }) =>  {
  return (
    <div className="wrapper">
      <div className='logo'>
        <img src={logo} className="App-logo" width={140} height={80} alt="logo" />
      </div>
      {children}
    </div>
  )
}

export default Wrapper