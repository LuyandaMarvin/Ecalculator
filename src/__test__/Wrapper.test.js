import React from "react";
import Wrapper from "../components/Wrapper"
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('<Wrapper />', () => {
  const wrapper = shallow(<Wrapper />);
  
    it('renders the correct html output', () => {
      expect(wrapper.html()).toMatchSnapshot();
    });
});
