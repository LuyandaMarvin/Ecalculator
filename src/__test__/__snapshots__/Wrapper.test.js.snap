// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> renders the correct html output 1`] = `"<div class=\\"wrapper\\"></div>"`;
